### PR TITLE
US001-view-job-roles

### PR DESCRIPTION
**Pull request by Cormac-F**
- Created jobService, jobController, Job and view-job files. 
- Shows a list of Job IDs and Job Names.

Screenshot 2023-12-07 at 11 25 26 am

### CODE REVIEW

File: views/list-jobs.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <title>Job Management System | Kainos</title>
+    <link rel="icon" href="../public/images/favicon.ico">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../public/css/styles.css">
+</head>
+
+    <nav class="navbar navbar-expand-lg bg-light sticky-top shadow-sm p-3 bg-white rounded">
+        <div class="container">
+            <a class="navbar-brand pb-2" href="./index.html">
+                <img src="../public/images/kainos-logo.png" alt="Kainos Logo" width="100" height="30">
```
Comment (jkelso662) [inline views/list-jobs.html:15]: Doesn't look like this is correctly referenced based on the screenshot on the ticket description. Also there is no `kainos-logo.png` image in this PR.

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Nice! Looks good but I think you have a missing image resulting in a broken image href. Fix that and this PR should be good :)

**Comment by CharlesM988** [comment]
Some changes to the styling of view-jobs.html.

- Modified view-jobs.html to include bootstrap styling inline with the rest of the pages
- Created styles.css and added styles


@jkelso662 

Screenshot 2023-12-12 at 10 26 31

**Review by amchugh689 — APPROVED** [review]
LGTM!

**Review by CharlesM988 — APPROVED** [review]
The issues with the logo were resolved. This should be merged now.

**Review by jkelso662 — APPROVED** [review]
Best to update the screenshot in the description with the fixed UI but I'll take your word for it that the image is fixed 😄

**Pull request merged**